pcl_ros/package.xml

```diff
@@ -10,10 +11,8 @@
 
   </description>
 
-  <author>Open Perception</author>
-  <author email="julius@kammerl.de">Julius Kammerl</author>
-  <author email="william@osrfoundation.org">William Woodall</author>
-
+  <maintainer email="julius@kammerl.de">Julius Kammerl</maintainer>
```

Comment:
I don't think either are maintainers, I'd just leave them as authors

Reply:
ament_xmllint complains about the author tag being nonexistant

Reply:
Probably good to leave ament_xmllint enabled.

Can do either:
1. leave them as maintainers
2. Add comment above the maintainers line to specify the original authors

Either way I was a little surprised to see that the author tag isn't part of the schema, maybe I'm doing something very wrong??

Reply:
Sorry for all the comments, I'm live streaming this thing apparently.

Looks like the author tag is listed as an optional element in schema v3. And so this is probably actually an ament_xmllint bug -- let me do some additional research to confirm and if so we can disable the xmllinting temporarily and leave the author tags.

https://www.ros.org/reps/rep-0149.html#package-format-3

Reply:
PEBKAC -- it was an ordering issue. Author tag must come after maintainer/license/url etc.

I've changed them back to authors and moved the tags down so ament_xmllint passes.